### ISSUE TITLE
Event/Timetable: long subject causes the start/end time of the appointment not to be displayed

### ISSUE THREAD

**Issue by marcschh** [indico/indico#4825]
**Describe the bug**
long subject causes the start/end time of the appointment not to be displayed

**To Reproduce**
1. Go to Category
2. Click on 'Create Event'
3. Click on 'Timetable'
3. Add new Entry with Subject of ~100 characters - 
4. See the Timetable Overiew - Entrys with such amount of characters does not show the Start/End Time

**Expected behavior**

**Screenshots**
If applicable, add screenshots to help explain your [problem.]
https://indico.mpi-magdeburg.mpg.de/event/12/timetable/#20210414.detailed

**Additional context**

**Comment by ThiefMaster** [comment]
I don't think this is something we can easily improve in the current timetable - in the end the space there is simply limited. We can't show a warning about this in management either since it depends on the available space how long the title can be before things get truncated..

We can look into improving the truncation logic when rewriting the timetable during a future version though. Ideally we always show the time and just truncate the title a bit, unless the space is extremely limited (e.g. due to many parallel blocks) and the time takes too much space compared to at least part of the title..

**Comment by marcschh** [comment]
` Ideally we always show the time and just truncate the title a bit, `
-> sounds great. and on mouse-over showing the full title
